types/jsdom/ts3.3/jsdom-tests.ts

```diff
@@ -0,0 +1,10 @@
+import '../jsdom-tests';
+import jsdom = require('jsdom');
+
+const dom = new jsdom.JSDOM();
+const domWindow = dom.window; // $ExpectType DOMWindow
+
+domWindow.document.querySelector('slot'); // $ExpectType HTMLSlotElement | null
+domWindow.AbstractRange.prototype; // $ExpectType AbstractRange
+domWindow.StaticRange.prototype; // $ExpectType StaticRange
+domWindow.CustomElementRegistry.prototype; // $ExpectType CustomElementRegistry
```

Comment:
This file should have the tests that were in `types/jsdom/jsdom‑tests.ts`.

Reply:
ts3.3 tests apply to 3.3 - 3.1, so should be moved from ts3.1. I missed deleting `ts3.1` initially, which made this NOT the case. It's fixed now.